components/02-molecules/banner/grand-hero/yds-grand-hero.twig

```diff
@@ -17,6 +17,7 @@
 {% set grand_hero__content__background = grand_hero__content__background|default('gray-800') %}
 {% set grand_hero__overlay_variation = grand_hero__overlay_variation|default('contained') %}
 {% set grand_hero__size = grand_hero__size|default('full') %}
+{% set video_background__button__background_color = grand_hero__content__background|default('gray-800') %}
```

Comment:
I think this variable should be set at the include level and not at the top here. Otherwise, you would need to add `video_background__button__background_color` as an available variable at the top of this file and that seems inconsistent with the other components.